_scripts/fetch-upstream-resources.sh

```diff
@@ -16,10 +17,15 @@ distribution_svn_branch="branches/${DISTRIBUTION_BRANCH}"
 if [ "${distribution_svn_branch}" = "branches/master" ]; then
 	distribution_svn_branch=trunk
 fi
+compose-cli_svn_branch="branches/main"
+if [ "${compose-cli_svn_branch}" = "branches/main" ]; then
+	compose-cli_svn_branch=trunk
+fi
```

Comment:
Perhaps use an underscore instead of a hyphen in the variable name;

```suggestion
compose_cli_svn_branch="branches/${COMPOSE_CLI_BRANCH}"
if [ "${compose_cli_svn_branch}" = "branches/main" ]; then
	compose_cli_svn_branch=trunk
fi
```

Reply:
updated